### ISSUE TITLE
Unhold not working in Chrome Version 61.0.3163.79 (Official Build) (64-bit) 

### ISSUE THREAD

**Issue by thuraucsy** [onsip/SIP.js#443]
I'm using freeswitch as server and using sipjs for _calling, answering, holding and unholding_.
Although **holding** work without any error, **unholding** is not working most of the time. Yes it works sometime if I unhold within 3 or 4 secs may be. I will paste the log in the following.

```

Working situation
Fri Sep 08 2017 12:56:23 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | RTCIceChecking Timeout Triggered after 5000 milliseconds
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:23 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer D expired for INVITE client transaction z9hG4bK7641051
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:24 GMT+0900 (Japan Standard Time) | sip.dialog | new UAC dialog created with status EARLY
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:37 GMT+0900 (Japan Standard Time) | sip.dialog | dialog 29amhb6dnf2hp2ta40vuegbn7tj4qjZHUt05t4rNc7N  changed to CONFIRMED state
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:37 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | stream added: VgKZOIOzWKA5zi8dFJ1Did6pZGNHqW7c
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:40 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | RTCIceGatheringState changed: complete
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:42 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | RTCIceChecking Timeout Triggered after 5000 milliseconds
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:51 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:52 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | stream removed: VgKZOIOzWKA5zi8dFJ1Did6pZGNHqW7c
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:55 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:55 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | stream added: VgKZOIOzWKA5zi8dFJ1Did6pZGNHqW7c
sip-0.7.8.js:2900 Fri Sep 08 2017 12:56:55 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer B expired for INVITE client transaction z9hG4bK5668212


Not working situation
Fri Sep 08 2017 12:57:22 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:23 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer B expired for INVITE client transaction z9hG4bK2815773
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:24 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer M expired for INVITE client transaction z9hG4bK2815773
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:25 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | stream removed: VgKZOIOzWKA5zi8dFJ1Did6pZGNHqW7c
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:27 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer B expired for INVITE client transaction z9hG4bK1308245
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:27 GMT+0900 (Japan Standard Time) | sip.transaction.ict | Timer M expired for INVITE client transaction z9hG4bK1308245
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:30 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:36 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:38 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media
sip-0.7.8.js:2900 Fri Sep 08 2017 12:57:39 GMT+0900 (Japan Standard Time) | sip.invitecontext.mediahandler | already have local media


```

For holding and unholding, both of the log first saying **already have local media** and then **stream removed** for holding and **stream added** for unholding. 

In not working situation, **already have local media** only show and no **stream removed** log is outputting.

**Comment by james-criscuolo** [comment]
Please try out [the PR branch](https://github.com/onsip/SIP.js/pull/426) which not only up-ends how we deal with media, but also adds official support for hold/unhold. The PR is close to complete, and will also mark a new version.

**Comment by egreenmachine** [comment]
We believe that this issue may be resolved in the latest tagged version [0.8.0](https://github.com/onsip/SIP.js/releases/tag/0.8.0). Can you please retest with that version and let us know if this is still an issue. We will close this issue on 11/1/17 if we do not hear back. Thanks.

**Comment by thuraucsy** [comment]
I retested it and unfortunately, I could not regenerate the **unholding** error again using SIP version 0.7.8. I could not think what cause that problem. Since, **unholding** works perfectly now, it is okay to close the issue. Thanks for the reply @james-criscuolo  and @egreenmachine.
